File: src/main/java/io/openfuture/openmessanger/web/request/GroupMessageRequest.java
package io.openfuture.openmessanger.web.request;

import io.openfuture.openmessanger.repository.entity.MessageContentType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class GroupMessageRequest {
    private String sender;
    private Integer groupId;
    private MessageContentType contentType;
    private String body;
}

File: src/main/kotlin/io/openfuture/openmessanger/web/request/GroupMessageRequest.kt
package io.openfuture.openmessanger.web.request

import io.openfuture.openmessanger.repository.entity.MessageContentType

data class GroupMessageRequest(
    val sender: String? = null,
    val groupId: Int? = null,
    val contentType: MessageContentType? = null,
    val body: String? = null,
    val attachments: List<Int> = emptyList()
)